test/test_art_scan.cpp

```
@@ -0,0 +1,575 @@
+// index.  Since the data only contains the ODD keys, you can probe
+// with EVEN keys and verify that the scan() is carried out from the
+// appropriate key in the data when the fromKey and/or toKey do not
+// exist in the data.
+//
+// @param fromKey
+//
+// @param toKey
+//
+// @param limit The largest key to be installed (if EVEN, then the
+// largest key is limit-1).
+template <typename TypeParam>
```

cppcheck (2.10-2)

[error] There is an unknown macro here somewhere. Configuration is required. If UNODB_DETAIL_DISABLE_CLANG_WARNING is a macro then please configure it.

(unknownMacro)